warning.rb

```diff
@@ -44,10 +44,16 @@ module Kernel
   # following categories:
   #
   # :deprecated :: Used for warning for deprecated functionality that may
-  #                be removed in the future.
+  #                  be removed in the future.
```

Comment:
This looks like it would result in misformatted documentation.

Reply:
Thanks. corrected.